### DISCUSSION TITLE
Append text to docx without using handlebars template?

### DISCUSSION THREAD

**Discussion by deasystephenj** [dolanmiu/docx#2276] [Q&A]
Is it possible to read an existing docx file, append some text to it, and then save as a new file without having the mustache template in the source file? It seems it should be possible as patching shows how to read/modify/save, but the modification is only applied to a specific piece of text that has been templatized, whereas I would like the ability to simply append text to the original doc, even if it is saved as a new doc.

**Comment by dolanmiu** [comment]
It would be possible if it was a feature, but it is not a feature im afraid, so therefore, currently, it is not possible
